Support repline 0.4 (#658)
jwiegley/hnix: README.md

$ ./dist/build/hnix/hnix --help
```

> ## Using REPL

Reviewer: ```suggestion
## Using the REPL
```

> To enter `hnix` REPL use

Reviewer: ```suggestion
To enter the `hnix` REPL use
```

```
hnix --repl
```

to evaluate an expression and make it available via REPL

> as `input` variable use

Reviewer: ```suggestion
To evaluate an expression and make it available in the REPL
as the `input` variable use
```

```
hnix --eval -E '(import <nixpkgs> {}).pkgs.hello' --repl
```

> Use `:help` command for list of all avaiable REPL commands.

Reviewer: ```suggestion
Use the `:help` command for a list of all available REPL commands.
```
